src/toil/cwl/cwltoil.py

```diff
@@ -1207,15 +1844,15 @@ def run(self, file_store: AbstractFileStore) -> Any:
             toil_get_file, file_store, index, existing
         )
 
-        # TODO: Pass in a real builder here so that cwltool's builder is built with Toil's fs_access?
-        #  see: https://github.com/common-workflow-language/cwltool/blob/78fe9d41ee5a44f8725dfbd7028e4a5ee42949cf/cwltool/builder.py#L474
-        # self.builder.outdir = outdir
-        # runtime_context.builder = self.builder
-
         process_uuid = uuid.uuid4()  # noqa F841
         started_at = datetime.datetime.now()  # noqa F841
 
-        logger.debug("Running CWL job: %s", cwljob)
+        original = cwltool.command_line_tool.check_adjust
+        def wrapper(builder, file_o):
+            original(builder, file_o)
+        cwltool.command_line_tool.check_adjust = wrapper
```

Comment:
This `original`/`wrapper` incantation should probably have a comment explaining itself.

Reply:
I've dropped this; I just had it so I could cram debugging in there.